web/app.js

```diff
@@ -158,7 +158,7 @@ const PDFViewerApplication = {
   baseUrl: "",
   _downloadUrl: "",
   _eventBusAbortController: null,
-  _windowAbortController: null,
+  _windowAbortController: new AbortController(),
```

Comment:
This could break third-party users of the viewer, since they probably expect being able to both add and remove window listeners which these changes will outright break (once removed there's no way back).

Given that you basically want something specifically for the mozcentral tests, can we please keep the `_windowAbortController` functionality as-is and introduce a new AbortController instead?
```suggestion
  _globalAbortController: new AbortController(),
  _windowAbortController: null,
```
You'd then pass the `_globalAbortController` to the various viewer-components and in `unbindWindowEvents` you'd do:
```javascript
if (AppOptions.get("isInAutomation")) {
  this._globalAbortController.abort();
}
```